src/renderer/screens/Welcome.js

```diff
@@ -125,7 +125,9 @@ class Welcome extends React.Component {
             <Button
               color="primary"
               variant="outlined"
-              onClick={navigate("/firmware-update")}
+              onClick={async () => {
+                navigate("/firmware-update");
```

Comment:
You added async here, but not the await.

Reply:
Nice catch, thank you! Fixed & force-pushed.